test/lib/BedrockTester.cpp

```diff
@@ -583,4 +583,36 @@ int BedrockTester::waitForPort(int port) {
     return 1;
 }
 
+STable BedrockTester::getStatus(bool control) {
+    // FYI: sometimes the status command doesn't return with every key/value expected
+    SData request("Status");
+    return SParseJSONObject(executeWaitVerifyContent(request, "200", control));
+}
+
+string BedrockTester::getStatusTerm(string term, bool control) {
+    STable responseTable;
+    int i = 0;
+
+    // FYI: sometimes the status command doesn't return with every key/value expected
+    // loop up to 1000 times until you find it
+    while (!responseTable.count(term) && i<1000) {
+        responseTable = getStatus(control);
+        i++;
+    }
+    SASSERT(i<1000);
```

Comment:
what is this assert for?

Reply:
to make it reliably fail if we hit 1000 attempts without getting the term we're looking for